lib/routes/retell.js

```diff
@@ -14,6 +15,11 @@ const service = ({logger, makeService}) => {
     const {from, to, direction, call_sid} = session;
     logger.info({session}, `new incoming call: ${session.call_sid}`);
 
+    /* Send ping to keep alive websocket as some platforms timeout, 25sec as 30sec timeout is not uncommon */
+    keepAlives[session.call_sid] = setInterval(() => {
```

Comment:
why is this an array?  Seems unnecessary and needlessly complicated unless I am missing something

Reply:
yes one websocket connection per call.  Doesnt mean we need that array though, a much simpler solution is possible

Reply:
moved it onto the session object